Client/Ecosia/UI/NTP/Cells/NTPImpactCell+Accessibility.swift

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+extension NTPImpactCell {
+    /// Accessibility Strings for impact cell texts that are NOT localized
```

Comment:
I agree with the fact that we should localize them. Wondering about the root cause of not having them localized in the first place? 🤔

Reply:
No idea, I thought this was just to prevent future delays since they could be volatile texts given the foreseen onboarding changes (namely the ones on this PR), and just kept it as it was.

But if you also think they should be localized I will go ahead and localize everything 🌏 🚀 😁

Reply:
I'm definitely onboarded with localizing them 💪